_includes/head.html

```diff
@@ -15,19 +15,32 @@
 
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
     <meta name="description" content="{{page.description}}">
-
-    <meta property="og:url" content="https://expressjs.com">
-    <meta property="og:type" content="website">
-    <meta property="og:title" content="{{page.title}}">
+    <meta property="og:url" content="https://expressjs.com{{page.url}}">
+    
+    {% if page.author %}
+    <meta property="og:type" content="article" >
+    <meta property="og:article:author" content="{{page.author}}">
+    {% else %}
+    <meta property="og:type" content="website" >
+    {% endif %}
+    
+    <meta name="title" property="og:title" content="{{page.title}}">
     <meta property="og:description" content="{{page.description}}">
-    <meta property="og:image" content="https://expressjs.com/images/og.png">                
-
+    {% if page.image %}
+    <meta property="og:image" content="{{page.image}}" > 
```

Comment:
I had been thinking about making the image dynamic, but Jekyll doesn’t allow it, or at least that's what I've been seeing. Maybe we could create dynamic OG images in Astro or Next.js, but we would need to do it in an external repository.

cc: @expressjs/docs-captains

Reply:
![Captura de Tela 2024-10-24 às 21 28 20](https://github.com/user-attachments/assets/65d576b5-061f-4b90-a652-768db0944742)
Eu estava testando e parece funcionar, o que você acha?

Reply:
** The image would be just for testing, if you like it, I will remove the image, then we just add the correct ones and it will work

Reply:
I like the idea of having the option to add images on each page if we want, although this would be a manual task, and I don’t think anyone would want to create images for every blog post

Reply:
+1 to the idea of having the option to add images